src/Repository/Util.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace PolderKnowledge\EntityService\Repository;
+
+use Doctrine\Common\Collections\Criteria;
+
+class Util
+{
+    public static function normalizeCriteria($criteria)
+    {
+        if ($criteria instanceof Criteria) {
+            return $criteria;
+        } elseif (is_array($criteria)) {
+            return self::createCriteriaFromArray($criteria);
+        } else {
```

Comment:
else in needless since the other branches are returning.

Reply:
I personally don't agree with this style because it does not work in programming languages where if is an expression.

But adjusted nonetheless.